migrations/migrations/0010_add_sequential_number_to_assignment.py

```diff
@@ -0,0 +1,46 @@
+from collections import defaultdict
```

Comment:
Please move all migrations into one. This means that you cannot use the base `AddFieldMigration`, but instead have to write the code yourself, but has the advantage that you can do all steps as one and can deduplicate the code.

Reply:
Done.